### PR TITLE
Changes to food hygiene qualification

### PR DESCRIPTION
**Pull request by gabimartini**
trello issue: https://trello.com/c/RrFI604
Screenshot 2022-05-03 at 16 54 08
6

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -34,10 +34,13 @@
     t.boolean "has_first_aid_training"
     t.date "first_aid_expires_at"
     t.string "qualifications", array: true
+    t.boolean "has_food_hygiene"
+    t.date "food_hygiene_achieved_on"
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
```
Comment (em-cd) [inline db/schema.rb:42]: This one has snuck in again! 😄

**Review by em-cd — CHANGES_REQUESTED** [review]
Nicely done @gabimartini!

The only thing which is missing from this is to add to the import script so that we set the new `has_food_hygiene` field based on whether the skills column in the CSV contains 'Food Hygiene'. Happy to chat about this if it's easier!

File: db/schema.rb (hunk 1 above)
Comment (gabimartini) [reply db/schema.rb]: :woman-facepalming: Fixed!

**Review by em-cd — APPROVED** [review]
LGTM! 🎉

File: db/schema.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabimartini) [reply db/schema.rb]: No way!!! I will remove now haha

[the remark this answers is not in the document]
Comment (gabimartini) [reply db/schema.rb]: done!

[the remark this answers is not in the document]
Comment (em-cd) [reply db/schema.rb]: How weird, I didn't write that message today but it posted it again 😂

**Pull request merged**